Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -74,8 +74,10 @@ public class CryptoStore: ObservableObject {
   private let txService: BraveWalletTxService
   private let ethTxManagerProxy: BraveWalletEthTxManagerProxy
   private let solTxManagerProxy: BraveWalletSolanaTxManagerProxy
+  let braveCore: BraveCoreMain?
```

Comment:
Are we able to store this as just the `IpfsAPI` instead of passing all of `BraveCoreMain` into Wallet?
Then I think we should be able to create a `MockIpfsAPI` instead of storing as optional throughout Wallet, and we can use the `MockIpfsAPI` in `previewStore`s 🙂.

Later we could possibly make a test version similar to `BraveWallet.TestKeyringService` / `BraveWallet.TestJsonRpcService()` / etc. This pattern would match newer test class setups on iOS cc @kylehickinson

Reply:
@StephenHeaps We don't really have a way to make `MockIpfsAPI` in this case, it isn't a mojo generated interface, but I agree we shouldn't pass `braveCoreMain`, just `ipfsAPI`

Reply:
This is not needed anymore